.github/workflows/render-readme.yaml

```
@@ -1,30 +1,33 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
 on:
   push:
-    paths:
-      - README.Rmd
+    paths: ['README.Rmd']
 
-name: Render README
+name: render-readme
 
 jobs:
-  render:
-    name: Render README
-    runs-on: macOS-latest
+  render-readme:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
     steps:
-      - uses: actions/checkout@v2
-      - uses: r-lib/actions/setup-r@v1
-      - uses: r-lib/actions/setup-pandoc@v1
-      - name: Install r2dii.data, rmarkdown, remotes, and the local package
-        run: |
-          install.packages("r2dii.data")
-          install.packages("remotes")
-          remotes::install_local(".")
-          remotes::install_cran("rmarkdown")
-        shell: Rscript {0}
-      - name: Render README
-        run: Rscript -e 'rmarkdown::render("README.Rmd")'
-      - name: Commit results
+      - name: Checkout repo
+        uses: actions/checkout@v3
```

```suggestion
        uses: actions/checkout@v4
```
https://github.com/actions/checkout/releases

---

Thank you
**shakes fist vaguely at r-lib**